tests/Demo.py

```diff
@@ -70,7 +70,7 @@ def local_main(config):
                 input_json_obj = json.loads(input_json_str1)
                 wo_id = hex(random.randint(1, 2**64 - 1))
                 input_json_obj["params"]["workOrderId"] = wo_id
-                input_json_obj["params"]["workerId"] = worker_obj.worker_id
+                input_json_obj["params"]["workerId"] = worker_id
```

Comment:
How is global variable "worker_id" getting initialized?

Reply:
It is initialized in workerRetrieve() API.

Reply:
Can we add a comment to specify this?

Reply:
Temporary fix. Manoj is removing this Demo.py file.

Reply:
Ok